server/encryptionkm/key_manager.go

```diff
@@ -124,7 +131,10 @@ func saveKeys(
 }
 
 // extractKeysFromKV unpack encrypted keys from etcd KV.
-func extractKeysFromKV(kv *mvccpb.KeyValue) (*encryptionpb.KeyDictionary, error) {
+func extractKeysFromKV(
+	kv *mvccpb.KeyValue,
+	helper ...keyManagerHelper,
```

Comment:
Can the length of `helper` only be 0 or 1?

Reply:
how can I enforce that?